test/ResourcesClassLibraryWithAttribute/ResourcesClassLibraryWithAttribute.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>Alternate.Namespace</RootNamespace>
```

Comment:
Is this change necessary?

`$(RootNamespace)` normally only controls the namespace of generated files such as `Resources.Designer.cs` or new files in the project. Since Localization uses only the resources and not the generated class, what matters here?

Reply:
This change is necessary. To prove it remove `[assembly: RootNamespace("Alternate.Namespace")]` from `ResourcesClassLibraryWithAttribute\AssemblyInfo.cs` in this PR and watch resources not be found. This is because dotnet uses RootNamespace as part of the name when automatically adding resources to the assembly, but `RootNamespace` is not available at runtime.

Reply:
Got it